R/Task_operators.R

```diff
@@ -470,14 +471,17 @@ getTaskWeights = function(task) {
 #' @export
 #' @examples
 #' task = makeClassifTask(data = iris, target = "Species")
-#' getTaskDictionary(task)
-getTaskDictionary = function(task) {
+#' getTaskDictionary(task, subset = NULL)
+getTaskDictionary = function(task, subset = NULL) {
+  getSubsetSize = function(subset) if (is.logical(subset)) sum(subset) else length(subset)
```

Comment:
There was a PR and we now have `checkTaskSubset`. This might want to be used here.